packages/core/test/insured-bridge/BridgeRouter.js

```diff
@@ -53,27 +74,69 @@ describe("BridgeRouter", () => {
         return ev.l2DepositContract === newDepositContract;
       });
     });
-    it("Add whitelisted Token mapping", async () => {
-      const l1Token = rando;
-      const l2Token = owner;
-      const bridgePoolAddress = l1CrossDomainMessenger;
-      assert(
-        await didContractThrow(
-          bridgeRouter.methods.whitelistToken(l1Token, l2Token, bridgePoolAddress).send({ from: rando })
-        ),
-        "OnlyOwner modifier not enforced"
-      );
+    describe("Whitelist tokens", () => {
+      it("Basic checks", async () => {
+        assert(
+          await didContractThrow(
+            bridgeRouter.methods
+              .whitelistToken(l1Token, l2Token, bridgePoolAddress, defaultGasLimit)
+              .send({ from: rando })
+          ),
+          "OnlyOwner modifier not enforced"
```

Comment:
do you want to include this `"OnlyOwner modifier not enforced"` no where else do we put errors on `asserts`.

Reply:
then again no downside in having it! I see you do it in a few places so it's ok to leave.

Reply:
Yeah I think its just more information which is not bad